docs/static/templates/document-planning-template.md

```diff
@@ -0,0 +1,57 @@
+---
```

Comment:
this feels like internal documentation...but I can be convinced it belongs here

Reply:
Yeah, I'm not sure about it. My argument for including it is that it's similar to some internal doc plan templates that we have, but it's simplified so that it can be used by developers for "self-serve" doc plans. If that's a use case we want to support, then I think we should include it. Otherwise, we can remove it from the doc along with the section on developing new topics, which incorporates it.

Reply:
I too wonder if this belongs here. Can we move this document (docs/static/templates/document-planning-template.md) to another issue or PR and further discuss it there?